### PR TITLE
Request to run 2014-10.  Minor script changes.

### PR DESCRIPTION
**Pull request by kdraeder**
pre_submit.csh
   Remove variable 'new_month' introduced for the push_prep.csh detour.
   Better quoting so that the git commands are not executed
   when the script is run.

purge.csh
   Add a line to remove the .h0. files of the purge month
   from the atm/hist directory.

repack_project.csh
   Comments raising the question of when to delete the original
   obs_seq_final files from $DOUT_S_ROOT/esp/hist.

repack_st_arch.csh
   Comments proposing a way to compress restart files that evade
   assimilate.csh's compression (when the first day of the month
   is a Monday).
   Added echos to explain some of the text output by the script.

### CODE REVIEW

File: repack_st_arch.csh [hunk 1]
```diff
@@ -472,9 +472,18 @@ if ($do_restarts == true) then
...
+#          set comp_arg = ''
+#          if ($day_o_month == 01) comp_arg = ' -z'
+#          ...
+#          echo " tar ${comp_arg} -c ...
+# This would still result in tar files with a different size 
+# for the first day of the month, but would save space (~67 Gb /month).
          while ($i <= $data_NINST)
             set INST = `printf %04d $i`
-            echo "tar -c -f ${yr_mo}/${data_CASE}.${INST}.alltypes.${rd}.tar "                     \
+            echo "tar -c -f ${yr_mo}/${data_CASE}.${INST}.alltypes.${rd}.tar "                 \
```
Comment (timhoar) [inline repack_st_arch.csh:486]: Remind me to discuss the advantages/disadvantages of   'tar -czvf' vs compressing the contents.

File: repack_st_arch.csh [hunk 2]
```diff
@@ -680,6 +689,7 @@ if ($do_history == true) then
       #            "TYPE_${INST}.eo " \
       set cmds_template = cmds_template_$models[$m]
 
+      echo "Checking existence of the template used to make a local cmds file."
```
Comment (timhoar) [inline repack_st_arch.csh:692]: I understand the template has to live somewhere ... why that location is being reported is confusing. I read the log file
and it says we are working in the lnd/hist directory but the reports stuff about cpl/hist ... and then follows it with 'ls' errors. Scary.

File: repack_st_arch.csh (hunk 1 above)
Comment (kdraeder) [reply repack_st_arch.csh:486]: I've made a note for Monday.

**Review by timhoar — APPROVED** [review]
These changes are fine.

**Pull request merged**